components/nimbus/android/src/main/java/org/mozilla/experiments/nimbus/Nimbus.kt

```diff
@@ -146,6 +167,18 @@ interface NimbusInterface : FeaturesInterface, GleanPlumbInterface {
      */
     fun setExperimentsLocally(payload: String) = Unit
 
+    /**
+     * This is functionally equivalent to a sequence of {setExperimentsLocally} the
+     * {applyPendingExperiments}.
+     *
+     * @param a `raw` resource identifier resolving to a JSON file downloaded from RemoteSettings
+     *       at build time.
+     * @return a Job. This may be cancelled, but only the loading from the resource will be cancelled.
+     *      If this is cancelled, then {initialize} is called, which copies the database in to an
+     *      in memory cache.
+     */
+    fun applyLocalExperiments(@RawRes file: Int): Job = Job()
```

Comment:
There were some concerns about expose more unnecessary public APIs earlier on. Since we as consumers do not use this API, it could be a private implementation. Will leave that up to you.

Reply:
RE the docs: it would be good to note that some tasks are guaranteed to happen (within the NonCancellableScope) which are not related to the `Job` returned.